### ISSUE TITLE
Unable to start Motorola Ready For/Smart Connect on a virtual display

### ISSUE THREAD

**Issue by fefo-dev** [Genymobile/scrcpy#5578]
## Environment

 - **OS:** Windows 10
 - **Scrcpy version:** 3.0
 - **Installation method:** portable release
 - **Device model:** Motorola G100 (nio)
 - **Android version:** 12

## Describe the bug

For those uninformed Motorola has a desktop environment known as Smart Connect (previously Ready For), enabled by connecting the phone on a monitor and controlled either by external hardware or app controls of its companion app: https://play.google.com/store/apps/details?id=com.motorola.mobiledesktop

I can use either scrcpy by itself, launch the virtual display, launch an app through the virtual display or capture the desktop environment while on a monitor by correctly enumerating the created display. However I can't start desktop mode solely with --new-display - it produces a black screen with no perceived control options.

Sadly I remember reading once that this mode checks if there's an video output to start, and some sort of dummy hdmi would be required. Had no luck searching for this info again.

Even if this is true, I wish to report this to find any possible fixes and workarounds. Been waiting for a feature like (https://github.com/Genymobile/scrcpy/pull/5370) for a long time and seeing it not work was quite disappointing.

Much appreciated as always.

**Comment by parkerlreed** [comment]
Yeah this is the same for Samsung. It won't activate DeX/ReadyFor with this method. As you mentioned you have to trigger it externally and then --diisplay-id to capture it.

I'm not aware of anything that can achieve this for either platform.
